source/clear-linux/get-started/virtual-machine-install/vmw-player.rst

```diff
@@ -140,12 +140,12 @@ Create and configure a new VM
 
    .. figure:: figures/vmw-player/vmw-player-05.png
       :scale: 100%
-      :alt: VMware Workstation 14 Player - Set disk capacity
+      :alt: VMware Workstation Player - Set disk capacity
 
-      Figure 5: VMware Workstation 14 Player - Set disk capacity
+      Figure 5: VMware Workstation Player - Set disk capacity
 
    .. note::
-      A minimal |CL| installation can exist on 600MB of drive space.
+      A minimal |CL| installation can exist on 600MB of drive space. For optimal performance with the |CL| Desktop image, we recommend 32GB of drive space.
```

Comment:
```suggestion
      For optimal performance with the |CL| Desktop image, we recommend 32GB of drive space.
```

While true in theory, this is actually not true anymore with our installers unfortunately. The ref to system-requirements is good and can provide more details for people looking for a minimal install

Reply:
Removed the 600MB, though I may take it as a personal challenge :)